### PR TITLE
gdk: trust nullability annotations

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Those were the ones that i noticed and needs fixes upstream
- gdk_content_deserializer_get_cancellable: needs a nullable annotation
- gdk_content_serializer_get_cancellable: needs a nullable annotation
- gdk_event_get_surface: needs a nullable annotation
- gdk_popup_get_parent: needs a nullable annotation

### CODE REVIEW

File: gdk4/Gir.toml [hunk 1]
```diff
@@ -243,27 +243,19 @@ generate_builder = false
...
-    name = "get_mime_type"
-        [object.function.return]
-        nullable = false
-    [[object.function]]
     name = "get_gtype"
     rename = "type_"
     [[object.function]]
+    name = "get_cancellable"
+        [object.function.return]
+        nullable = true # TODO: fixme upstream
```
Comment (sdroege) [inline gdk4/Gir.toml:251]: Please create an issue before merging

Comment (sdroege) [reply gdk4/Gir.toml:251]: Also for the others

**Comment by sdroege** [comment]
Looks generally correct otherwise

File: gdk4/Gir.toml (hunk 1 above)
Comment (bilelmoussaoui) [reply gdk4/Gir.toml:251]: Sent a PR https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4310

Comment (sdroege) [reply gdk4/Gir.toml:251]: Can you put a link to that into `Gir.toml`?

Comment (bilelmoussaoui) [reply gdk4/Gir.toml:251]: I don't plan on merging it yet, still working on the gtk ones

**Comment by bilelmoussaoui** [comment]
Okay, I have dropped all the `trust_return_value_nullability = false`, regenerated type by type; checked the corresponding C function and updated https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4310 with whatever fixes that are needed.

Once that merge request is merged, we can merge this one & drop the manual annotations here once we get a new gir file.

**Comment by bilelmoussaoui** [comment]
The merge request got merged, I went ahead and done something similar for the constructors as well https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4315. This should be ready to be merged.

**Pull request merged**